src/main/java/uk/co/ramp/people/Person.java

```diff
@@ -112,33 +119,34 @@ public void randomExposure(int t) {
         if (status == SUSCEPTIBLE) {
             LOGGER.info("Person with id: {} has been randomly exposed at time {}", getId(), t);
             updateStatus(EXPOSED, t);
+        } else {
+            String message = String.format("The person with id: %d should not be able to transition from %s to %s", id, status, EXPOSED);
+            LOGGER.error(message);
+            throw new InvalidStatusTransitionException(message);
         }
     }
 
     int getDistributionValue(double mean, ProgressionDistribution p) {
```

Comment:
I don't understand the `getDistributionValue` method.  The value of `sample` appears to be unused?  (And then '14' seems to be a magic number.)

Reply:
sample is a random variable taken from the chosen distribution (Gaussian, Exp etc) which is then rounded to an int, but with a maximum duration of 14 days. This is a magic number but I'll add a todo because its something I need to discuss with Sibylle and Louise.